src/transformers/pipelines.py

```diff
@@ -820,7 +820,7 @@ def __init__(
 
         self.topk = topk
 
-    def ensure_exactly_one_mask_token(self, masked_index: Union[torch.Tensor, np.ndarray]):
```

Comment:
You can use "quotes" here to prevent imports, that's what we do elsewhere

Reply:
Didn't know this one! Thanks 🙏